test/Esprima.Tests.Test262/Test262Harness.settings.json

```diff
@@ -5,8 +5,8 @@
   "Namespace": "Esprima.Tests.Test262",
   "Parallel": true,
   "ExcludedFeatures": [
-    "regexp-v-flag",
-    "import-assertions"
+    "import-assertions",
```

Comment:
should this exclude be removed if this is the feature that should be parsed? might also be worth use the latest commit sha from test262 repo for correct input

Reply:
The `import-assertions` feature is for the old version of the proposal. We doesn't support that, so it should be excluded.

Unfortunately, [the updated tests](https://github.com/tc39/test262/pull/3843) haven't landed in the test262 repo yet.

What do you think, how should we proceed? Shall we wait with merging our PR until the updated tests get merged in test262?

Reply:
I think merge is fine as we can always add tests later/enable feature and fix possible regressions 👍🏻

Reply:
Ok, then I'll go ahead. Also, I've subscribed to the test262 PR and will check those tests as soon as they become available.